React/Views/RCTWrapperViewController.h

```diff
@@ -11,6 +11,6 @@
 
 @interface RCTWrapperViewController : UIViewController
```

Comment:
I think this is one of the few places where RN uses `UIVIewController` which isn't handled by our shim AFAIK.

I think this is OK to leave though.. iirc, this was an experimental control to get autolayout to work with yoga.